acapi/resources/acquiadata.py

```diff
@@ -90,16 +94,23 @@ def request(self, uri=None, method='GET',
         if uri is None:
             uri = self.uri
 
-        headers = {'User-Agent': self.USER_AGENT}
+        user_agent_header = {'User-Agent': self.USER_AGENT}
+        if headers:
```

Comment:
`headers` are defined as an empty `dict` by default. Why not use `headers['User-Agent'] self.USER_AGENT` here instead of the `if` logic?